sass/includes/_filters.scss

```diff
@@ -0,0 +1,127 @@
+.filters {
+    $root: &;
+    display: flex;
+    gap: 2rem;
+    padding: 1.5rem 0;
+    flex-direction: column;
+
+    @media only screen and (min-width: #{$screen__md + 1px}) {
```

Comment:
Could we import the new media queries (`@use "@nationalarchives/frontend/nationalarchives/tools/media";`) and replace these rules with queries like `@include media.on-mobile {}`?

The full list can be found here: https://github.com/nationalarchives/tna-frontend/blob/main/src/nationalarchives/tools/_media.scss#L3-L15

Reply:
Unfortunately it looks like that list doesn't have quite the same granularity which was useful for the filters - it was hard to fit them all across but I also didn't want to leave them stacked for too long, so I had a spot between 992px and 1200px where they sat horizontally but with a smaller gap.

Is there any scope, if we use the `tna-frontend` media queries, for a more granular custom breakpoint in between? If not I'll need to rejig the layout a bit, perhaps having the fields 2-across at medium breakpoints.

Reply:
Okay, let's keep it for now. Ideally we wouldn't have too many breakpoints as there then becomes a lot more scenarios to test but we might have to think about adding a bit more flexibility in the future.